tq/transfer_queue.go

```diff
@@ -307,7 +307,7 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 	}
 
 	q.useAdapter(bRes.TransferAdapterName)
-	q.startProgress.Do(q.meter.Start)
```

Comment:
Seems legit. I went ahead and added some tests in the `progress` package to assert that `Start()` and `Pause()` can be called multiple times.